serverless/src/getConceptUpdatesReport/__tests__/handler.test.js

```diff
@@ -212,13 +245,24 @@ describe('createCsvReport', () => {
         userId: null,
         entity: undefined,
         operation: 'Update',
+        systemNote: null,
```

Comment:
can you also include a test where date is null for the change note item?   Just to make sure nothing breaks there.